client/lib/plans/features-list.js

```diff
@@ -856,65 +856,65 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_SECURITY_SCANNING_JETPACK ]: {
 		getSlug: () => constants.FEATURE_SECURITY_SCANNING_JETPACK,
-		getTitle: () => i18n.translate( 'Advanced Security' ),
+		getTitle: () => i18n.translate( 'Advanced security' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Advanced Security:' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)